Framework/DataHandling/src/LoadILLSANS.cpp

```diff
@@ -256,12 +256,13 @@ LoadILLSANS::loadDataIntoWorkspaceFromMonitors(NeXus::NXEntry &firstEntry,
         positionsBinning.push_back(static_cast<double>(i));
 
       // Assign X
-      m_localWorkspace->dataX(firstIndex)
+      m_localWorkspace->mutableX(firstIndex)
           .assign(positionsBinning.begin(), positionsBinning.end());
       // Assign Y
-      m_localWorkspace->dataY(firstIndex).assign(data(), data() + data.dim2());
+      m_localWorkspace->mutableY(firstIndex)
+          .assign(data(), data() + data.dim2());
       // Assign Error
-      MantidVec &E = m_localWorkspace->dataE(firstIndex);
+      auto &E = m_localWorkspace->mutableE(firstIndex);
       std::transform(data(), data() + data.dim2(), E.begin(),
                      LoadHelper::calculateStandardError);
```

Comment:
Is this just computing the `sqrt` of the counts? If so, this could be done internally automatically by `Histogram`:

```cpp
m_localWorkspace->Histogram(firstIndex, BinEdges(vectorSize, LinearGenerator(0.0, 1.0)), Counts(data(), data() + data.dim2()); // NOTE: no E specified, so it defaults to sqrt(Counts)
```